### PR TITLE
New package: MacPawWayLTD.ClearVPN version 3.1.1

### PR DESCRIPTION
**Pull request by globart**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/188367)

### CODE REVIEW

**Comment by globart** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 897.)

File: manifests/m/MacPawWayLTD/ClearVPN/3.1.1/MacPawWayLTD.ClearVPN.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+PackageIdentifier: MacPawWayLTD.ClearVPN
+PackageVersion: 3.1.1
+InstallerType: exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- Architecture: x86
```
Comment (Trenly) [inline manifests/m/MacPawWayLTD/ClearVPN/3.1.1/MacPawWayLTD.ClearVPN.installer.yaml:12]:
```suggestion
- Architecture: x64
```

image

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Verify:
image
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**